docs/content/latest/api/ycql/ddl_create_table.md

```diff
@@ -52,12 +52,12 @@ partition_key_column_list ::= '(' column_name [ ',' column_name ...] ')' | colum
 
 clustering_key_column_list ::= [ ',' column_name ...]
 
-table_option ::= WITH table_property [ AND table_property ...]
-
 table_property ::= { property_name = property_literal
                      | CLUSTERING ORDER BY '(' column_ordering_property [ ',' column_ordering_property ...] ')'
                      | COMPACT STORAGE }
 
+table_options = table_property [AND table_property ...]
```

Comment:
We should change this to:
table_properties = [table_options]
                                  [[AND] CLUSTERING ORDER BY '(' column_ordering_property [ ',' column_ordering_property ...] ')']
                                  [[AND] COMPACT STORAGE]

table_options = property_name '=' property_literal [AND property_name '=' property_literal ...]

See the Cassandra docs for example - https://docs.datastax.com/en/dse/5.1/cql/cql/cql_reference/cql_commands/cqlCreateTable.html

Reply:
Pushed changes

Reply:
@ndeodhar is it ok after latest commit ?